libs/go/libp2p_node/dht/dhtnode/acn_message.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
```

Comment:
Why in two places? Have it in one place only please! (acn package)

Reply:
ok